proof/courier.go

```diff
@@ -332,9 +333,16 @@ func (h *HashMailCourier) DeliverProof(ctx context.Context, addr address.Taro,
 
 	// We'll wait to receive the ACK from the remote party over their
 	// stream.
-	log.Infof("Waiting for receiver ACK via sid=%x", receiverStreamID)
-	if err := h.mailbox.RecvAck(ctx, receiverStreamID); err != nil {
-		return err
+	timeout := time.Second * 5
```

Comment:
This should probably be a package level var/constant to make it easy to make it configurable in the future.

Reply:
Added a config field to the hashmail courier config. Let me know if you'd like it more global than that.